packages/apps/human-app/frontend/src/modules/worker/profile/components/identity-verification-control.tsx

```diff
@@ -1,21 +1,37 @@
 import { useTranslation } from 'react-i18next';
+import { Stack, Typography } from '@mui/material';
+import { Chip } from '@/shared/components/ui/chip';
 import { useWorkerIdentityVerificationStatus } from '../hooks';
+import { KycStatus } from '../types';
```

Comment:
```suggestion
import { KycStatus } from '../types';
```
Let's rename it to `IdvStatus`